### PR TITLE
Hacktoberfest landing

### PR DESCRIPTION
**Pull request by ruskibenya**
## Description

Updates to the Hacktoberfest Landing Page:
- Suggested Repos
- Event Cards
- FAQ
- Featured Talk Card
- Stylings

## Deploy Notes

Notes regarding deployment the contained body of work. These should note any db migrations, etc.

### CODE REVIEW

File: custom/landing_pages/hacktoberfest.yml [hunk 1]
```diff
@@ -27,132 +31,514 @@ page:
...
-              <h2 class="Vlt-black"><b>Talks</b></h2>
+              <h2 class="Vlt-black"><b>Featured Talks</b></h2>
               <div class="Vlt-grid Vlt-grid--stack-flush">
-                <a href="https://www.digitalocean.com/community/tech_talks/level-up-as-an-open-source-contributor" target="_blank" class="Vlt-col Vlt-col--1of3 talks">
-                  <img src="https://res.cloudinary.com/dsg3ucw55/image/upload/v1599751033/lorna-hacktoberfest_w579fk.png" alt="Hacktoberfest Level Up as asn Open Source Contributer Lorna Mitchell">
+                <a href="https://organize.mlh.io/participants/events/4525-acm-juit-open-source-season-hacktoberfest-meetup" target="_blank" class="Vlt-col Vlt-col--1of2 talks">
+                  <div class="Vlt-card talk2-card">
+                    <div class="Vlt-card__content Vlt-grid">
+                      <div class="Vlt-col Vlt-col--1of3 Vlt-center vertical-align">
+                        <img class="Vlt-center avatar" src="https://res.cloudinary.com/dsg3ucw55/image/upload/v1600853933/beng3_n3mxpj.jpg" alt="Garann Means"><img>
```
Comment (bencgreenberg) [inline custom/landing_pages/hacktoberfest.yml:374]:
```suggestion
                        <img class="Vlt-center avatar" src="https://res.cloudinary.com/dsg3ucw55/image/upload/v1600853933/beng3_n3mxpj.jpg" alt="Ben Greenberg"><img>
```

File: custom/landing_pages/hacktoberfest.yml [hunk 2]
```diff
@@ -27,132 +31,514 @@ page:
...
+                        <h3 class="Vlt-white Vlt-center speaker"><b>Ben Greenberg</b></h3>
+                        <h5 class="Vlt-white Vlt-center italic">Ruby Developer Advocate</h5>
+                      </div>
+                      <div class="Vlt-col Vlt-col--2of3 vertical-align">
+                        <h3 class="Vlt-white Vlt-center">The Benefits of Participating in<br> The Open Source Community</h3>
+                        <h3 class="Vlt-white Vlt-center"><b>Thursday, October 1</b></h3>
+                        <h5 class="Vlt-white Vlt-center">5:30 EDT / 21:30 UTC</h5>
```
Comment (bencgreenberg) [inline custom/landing_pages/hacktoberfest.yml:381]:
```suggestion
                        <h5 class="Vlt-white Vlt-center">5:30 IST / 12:00 UTC</h5>
```

Comment (ruskibenya) [reply custom/landing_pages/hacktoberfest.yml:381]: Toda Achi

**Review by bencgreenberg — APPROVED** [review]
Visually inspected, looks good to me!

**Pull request merged**